packages/client/hmi-client/src/components/dataset/Dataset.vue

```diff
@@ -2,83 +2,129 @@
 	<section class="asset">
 		<header>
 			<div class="simulation" v-if="dataset?.simulation_run">Simulation run</div>
-			<h4 v-html="dataset?.name" />
+			<h4 v-html="dataset?.name" class="model-name" />
```

Comment:
You can change the style of h4 in `asset.scss` and remove the class names from the h4s in Document.vue, Model.vue, Dataset.vue, etc.

Reply:
Possibly. I'm not sure what that will do to other pages / components.

Reply:
Any component with the class `"asset"` would be affected.